Bugsnag/Storage/BSGFileLocations.m

```diff
@@ -19,39 +22,44 @@ static BOOL ensureDirExists(NSString *path) {
     return YES;
 }
 
-static NSString *rootDirectory(NSString *fsVersion) {
+static NSString *rootDirectory(NSString *fsVersion, NSString *subdirectory) {
     // Default to an unusable location that will always fail.
     static NSString* defaultRootPath = @"/";
     static NSString* rootPath = @"/";
-
-    static dispatch_once_t onceToken;
-    dispatch_once(&onceToken, ^{
```

Comment:
As mentioned previously, we gotta loose the dispatch once. I don't think it is a performance problem. Results in some whitespace changes.